src/app/features/characters/components/character-info/character-info.component.ts

```diff
@@ -9,17 +10,16 @@ import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
   styleUrl: './character-info.component.scss',
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class CharacterInfoComponent {
-  @Input() character!: any;
-  name = '';
-  url = '';
-  info = '';
-  comics: any = [];
+export class CharacterInfoComponent implements OnChanges {
+  @Input() character!: MarvelChar | null;
+
+  url!: string;
+
+
+  ngOnChanges(): void {
```

Comment:
it is possible to implement in the `ngOnInit` like in `character-card.component.ts`, isn't it?

Reply:
It will be changed in the next PR